ddp-workspace/projects/ddp-atcp/app.yaml

```diff
@@ -0,0 +1,13 @@
+service: osteo
```

Comment:
Here should be `atcp`, please, fix.
Also, you need to update circleci config - https://github.com/broadinstitute/ddp-angular/blob/develop/.circleci/config.yml#L360
You should add new study_key, study_guid and add new jobs for `atcp` study